PennMobile/Setup + Navigation/PennMobile.swift

```diff
@@ -14,7 +14,8 @@ struct PennMobile: App {
     @UIApplicationDelegateAdaptor var delegate: AppDelegate
     @ObservedObject var authManager = AuthManager()
     @ObservedObject var homeViewModel = StandardHomeViewModel()
-    
+    @StateObject var popupManager = PopupManager()
```

Comment:
`PopupManager` should be declared on `RootView` and not the `App` struct itself - declaring them here will mean every window of the app (on iPadOS or macOS) shares the same popup state.